portfolio/src/main/webapp/style.css

```diff
@@ -1,19 +1,54 @@
+body {
+  background-color: #292929;
+}
+
 #content {
-  margin-left: auto;
   margin-right: auto;
   width: 100%;
 }
 
+/* Header */
+#header_container {
+  background-color: rgba(1, 2, 3, 0.7);
+  border-bottom: 1px solid black;
+  overflow: hidden;
+}
+
+#header_title {
+  width: calc(50% - 1.5em);
+  margin-left: 1em;
+  float: left;
+  color: rgba(236, 233, 227, 1);;
+}
+
+#media_wrapper {
+  text-align: right;
+}
+
+#header_media {
+  width: calc(50% - 1.5em);
+  display: inline-block;
+  margin-right: 1em;
+}
+
+.nav_media li{
+  display: inline-block;
+  align-content: flex-end;
+}
+
 /* Peronsal and Profesional Information */
 
 #information_container {
+  background-color: rgba(236, 233, 227, 1);
```

Comment:
I'm no CSS expert, but I see this color repeated. Is there a way to externalize it so that if you want to update the color, you only need to change one spot? Something like:

myColor = rgba(236, 233, 227, 1);
...information_container {
    background-color: myColor
}

Reply:
Just learned that was possible! Thank you for the recommendation Mark!